docs/concepts/trading-framework/fees.md

```diff
@@ -34,7 +34,7 @@ The aggressor *buying* the base asset for the quote asset pays more in their tra
 ### Fee distribution and breakdown
 Fees are calculated when a trade is filled, and paid in the market's settlement currency. The fees due are taken from the collateral in the trader's general account. 
 
-The fee is divided between the maker for the trade, the infrastructure providers, and the liquidity provider(s) for each market.
+If each fee type is set to have a value, the fee is divided between the maker for the trade, the infrastructure providers, the liquidity provider(s) for each market, and the network's treasury.
```

Comment:
I think "the network's treasury" is fine for collectively referring to the treasury and buyback fee components but raising just in case you don't and want to add the buyback here.